docs/tech/python-packages.md

```diff
@@ -0,0 +1,44 @@
+# Python packages
+
+A lot of our code is published as Python packages on [PyPI][].
+
+[PyPI]: https://pypi.org/
+
+## Development
+
+Make changes to the repository that contains the Python package using
+the normal GitHub flow.
```

Comment:
This is slightly confusing because there is also git flow, the tool. The tagging and version bump below can be done using `git flow release` which is probably what we would recommend in most cases (I think).